### DISCUSSION TITLE
reference for storing a bitmap in entity framework sqlite

### DISCUSSION THREAD

**Discussion by SirZeeno** [AvaloniaUI/Avalonia#16291] [Q&A] [answered]
Is there a way I can reference a bitmap in entity framework without having to map the entire bitmap functions and constructors? Because I can't map any constructors in entity framework and just want a simple value I can store to recall it later on, without having to pull the file from the web again and convert it. any help in this would be greatly appreciated :)

**Comment by maxkatz6** [comment, marked as answer]
You can store a byte array or Stream in the database. You can use these to create bitmap when you need it.

**Reply by SirZeeno** [reply]
Yes, I remembered that after taking a small break and coming back to it. Thank you very much though :)
